### PR TITLE
feat(trim_galore): updates trim_galore, adds sif

### PR DESCRIPTION
**Pull request by jemten**
### This PR fixes:

- see above

### How to test:

- Automatic and continuous test pass

### Expected outcome:
- Installation, unit and integration tests pass

### Review:
- [ ] Code review
- [ ] New code is executed and covered by tests
- [ ] Tests pass

### CODE REVIEW

File: lib/MIP/Get/Executable.pm [hunk 1]
```diff
@@ -241,6 +241,11 @@ q?'my ($version) = /\(htslib\)\s+(\S+)/xms; if($version) {print $version;last;}'
             version_regexp =>
               q?'my ($version) = /TIDDIT-(\S+)/xms; if($version) {print $version;last;}'?,
         },
+        trim_galore => {
```
Comment (henrikstranneheim) [inline lib/MIP/Get/Executable.pm]: Nice!

Comment (henrikstranneheim) [reply lib/MIP/Get/Executable.pm]: Maybe add "pigz" as well?

Comment (jemten) [reply lib/MIP/Get/Executable.pm]: Good idea

Comment (jemten) [reply lib/MIP/Get/Executable.pm]: No that I think about it. Should we log the version of these helper programs? It's a bit like logging the version of bzip2

Comment (henrikstranneheim) [reply lib/MIP/Get/Executable.pm]: I would not say that it is super critical. If it easy - consider doing it. I am for instance going to skip this with bwakit as it does not output a version in any way.

**Pull request merged**